test/testCLI.ts

```diff
@@ -78,4 +78,34 @@ export function run() {
     t.snapshot(readFileSync('./ReferencedType.d.ts', 'utf-8'))
     unlinkSync('./ReferencedType.d.ts')
   })
+
+  test('files in (-i), files out (-o)', t => {
+    execSync(
+      'node dist/src/cli.js -i ./test/resources/MultiSchema/**/*.json -o ./test/resources/MultiSchema/out'
+    ).toString()
```

Comment:
Looks like you're mimicking the old test cases here, but I think you can safely remove the `.toString()` here and elsewhere right?